sale_exception/wizard/sale_exception_confirm.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2011 Akretion, Camptocamp, Sodexis
+# Copyright 2018 Akretion, Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import api, fields, models
+
+
+class SaleExceptionConfirm(models.TransientModel):
+    _name = 'sale.exception.confirm'
+    _inherit = ['exception.rule.confirm']
+
+    related_model_id = fields.Many2one('sale.order', 'Sale')
+
+    @api.multi
+    def action_confirm(self):
+        self.ensure_one()
```

Comment:
Question, `self.ensure_one()` in multi ? Do we want to be able to call the wizard on multiple SO ?

Reply:
It's a button on a wizard: you won't use it on multiple SO and in any case you can relate only one SO per wiz, so... :)

Reply:
It's upstream API https://github.com/OCA/server-tools/blob/11.0/base_exception/wizard/base_exception_confirm.py#L38